faculty_mill/faculty_reporter.py

```diff
@@ -9,9 +8,9 @@
 import click
 import sherlockml
 import sml.auth
-from papermill.cli import papermill
 
 from .version import print_version
+from .execute import run
```

Comment:
I think the concensus is that `faculty_mill.execute` is better than using the relative import (although I'm not sure why).

Reply:
I would be fairly keen to keep the relative import because it's nicer

Reply:
Fine. In that case I think there was somewhere else that used the `faculty_mill.` style you might want to change for consistency

Reply:
yep, will do!